bolt-jakarta-jetty/src/test/java/samples/EventsApp.java

```diff
@@ -0,0 +1,44 @@
+package samples;
+
+import com.slack.api.bolt.App;
+import com.slack.api.bolt.jakarta_jetty.SlackAppServer;
```

Comment:
This line of code is the only difference from the existing way using jetty 9.x (=javax.servlet compatible version).